<?xml version="1.0"?>
<launch>
    <arg name="is_bag" default="true"/>
    <arg name="bag_file" default="$(find experiment_analyzer)/world/good_sc1.bag"/>
    <arg name="rviz_display" default="true"/>
    <arg name="use_sim_time" default="false"/>
    <group if = "$(arg is_bag)">
        <node name="ros_bag_play" pkg="rosbag" type="play" args="--clock -r 1.0 -s 0 $(arg bag_file)" >
        </node>
    </group>
    <group if = "$(arg rviz_display)">
        <node name="rviz" pkg="rviz" type="rviz" args="-d $(find experiment_analyzer)/launch/rviz_config/abc.rviz" output="log"/>
    </group>

    <node pkg="experiment_analyzer" type="interpreter" name="interpreter" output="screen">
        <param name="global_frame_id" value="map"/>
        <param name="optical_frame_id" value="zed2_left_camera_optical_frame"/>
        <param name="base_frame_id" value="base_link"/>
    </node>

</launch>